### PR TITLE
PR to add README.md for better documentation

### PR DESCRIPTION
**Pull request by souvikmishra**

### CODE REVIEW

File: src/lib/utils/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        ├── oysterServices.ts
+        └── receiverRewardServices.ts
+```
+
+## 📖 Description
+
+- `abis/`: This folder contains all the contract ABIs that are used in the project.
+  Each file contains the ABI of a single contract. For example, `bridge.ts` contains the ABI of the `Bridge` contract. These are used by the contract controller of their respective contract.
+- `constants/`: This folder contains all the constants that are used inside the project. Each file contains the constants of a single type. For example, `bridgeConstants.ts` contains the constants related to the `Bridge` route. As a rule of thumb if something is not going to change in the future and does not depend on which chain the user is on, or is a default value it should reside here.
+- `data-modifiers/`: This folder contains all the modifiers that are used to modify raw data into a more structured format . Each file contains the modifiers of a single type. For example, `oysterModifiers.ts` contains the modifiers related to the `Oyster` route. Modifiers may introduce new properties to the data inorder to make it easier to cosume.
```
Comment (prateekreddy) [inline src/lib/utils/README.md:51]: typo cosume -> consume

File: src/lib/utils/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- `helpers/`: This folder contains all the helper functions that are used inside the project. These are functions/logic that are being used in multiple places and are grouped by the common use case they adher to. For example, `conversionHelpers.ts` contains the helpers related to conversion of data from one format to other.
+- `services/`: This folder making contract calls and updating the local state of the app simultaneously. Each file contains the services of a single type. For example, `oysterServices.ts` contains the services related to the `Oyster` route.
```
Comment (prateekreddy) [inline src/lib/utils/README.md:53]: It isn't very clear what "services of a single type" means

File: src/lib/types/environmentTypes.ts [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
-	block_explorer_name: string;
-	block_explorer_url: string;
-	rpc_url: string;
-	backend_network_id: string;
-	contract_addresses: ContractAddress;
-	subgraph_urls: SubgraphUrls;
-	oyster_token: keyof Tokens;
-	oyster_rate_metadata: OysterRateMetadata;
-	tokens: Tokens;
+	chain_id: string; // hex string i.e. for arbitrum goerli: '0x66eed'
```
[the remark this answers is not in the document]
Comment (souvikmishra) [reply src/lib/types/environmentTypes.ts:15]: nope, not as of now

File: src/lib/utils/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    └── services/
+        ├── oysterServices.ts
+        └── receiverRewardServices.ts
+```
+
+## 📖 Description
+
+- `abis/`: This folder contains all the contract ABIs that are used in the project.
+  Each file contains the ABI of a single contract. For example, `bridge.ts` contains the ABI of the `Bridge` contract. These are used by the contract controller of their respective contract.
+- `constants/`: This folder contains all the constants that are used inside the project. Each file contains the constants of a single type. For example, `bridgeConstants.ts` contains the constants related to the `Bridge` route. As a rule of thumb if something is not going to change in the future and does not depend on which chain the user is on, or is a default value it should reside here.
```
[the remark this answers is not in the document]
Comment (souvikmishra) [reply src/lib/utils/README.md:50]: that would reside inside the config of that chain

**Review by prateekreddy — APPROVED** [review]

**Pull request merged**